script/check-env.sh

```diff
@@ -2,18 +2,30 @@
 
 set -e # Abort on error
 
+RAILS_ENV=$(grep RAILS_ENV .env | xargs)
```

Comment:
Le variabili sono già state prese in `docker-run.sh:12`, in teoria sono accessibili anche qua dentro

Reply:
Hai ragione  🙄 
Mi ero perso quella parte